storybook/stories/Examples/Pie/PieWithNeedle.stories.tsx

```diff
@@ -71,6 +81,7 @@ export const PieWithNeedle = {
             fill="none"
             activeShape={Needle}
           />
+          <Tooltip defaultIndex={1} />
         </PieChart>
```

Comment:
I guess I do think it's kind of weird we're using tooltip to set the active shape state. Not a blocker but feel like it's not that intuitive

Reply:
Yep I was wondering the same. I think the cause is the new methods for registering tooltipEventType - the new logic depends on tooltip being present in the DOM. Which will change later.